compiler/src/iree/compiler/Codegen/Passes.h

```diff
@@ -445,6 +445,8 @@ void addGPUTransformDialectPasses(OpPassManager &pm);
 /// module-level pass manager.
 void addGPUSimpleDistributePassPipeline(OpPassManager &pm);
 
+void addGPUDataTilingPasses(OpPassManager &pm);
```

Comment:
can you rename that one too?

Reply:
oops, I did not find it because I only searched for `LLVMGPUDataTiling`. Good catch; fixed!